integ-test/src/test/java/org/opensearch/sql/sql/SimpleQueryStringIT.java

```diff
@@ -32,43 +31,52 @@ public void init() throws IOException {
 
   @Test
   public void test_mandatory_params() throws IOException {
-    String query = "SELECT Id FROM "
-        + TEST_INDEX_BEER + " WHERE simple_query_string([\\\"Tags\\\" ^ 1.5, Title, 'Body' 4.2], 'taste')";
+    String query =
+        "SELECT Id FROM "
+            + TEST_INDEX_BEER
+            + " WHERE simple_query_string([\\\"Tags\\\" ^ 1.5, Title, 'Body' 4.2], 'taste')";
     var result = new JSONObject(executeQuery(query, "jdbc"));
     assertEquals(16, result.getInt("total"));
   }
 
   @Test
   public void test_all_params() throws IOException {
-    String query = "SELECT Id FROM " + TEST_INDEX_BEER
-        + " WHERE simple_query_string(['Body', Tags, Title], 'taste beer', default_operator='or',"
-        + "analyzer=english, analyze_wildcard = false, quote_field_suffix = '.exact',"
-        + "auto_generate_synonyms_phrase_query=true, boost = 0.77, flags='PREFIX',"
-        + "fuzzy_transpositions = false, lenient = true, fuzzy_max_expansions = 25,"
-        + "minimum_should_match = '2<-25% 9<-3', fuzzy_prefix_length = 7);";
+    String query =
+        "SELECT Id FROM "
+            + TEST_INDEX_BEER
+            + " WHERE simple_query_string(['Body', Tags, Title], 'taste beer',"
+            + " default_operator='or',analyzer=english, analyze_wildcard = false,"
```

Comment:
And here

Reply:
Don't believe we can, as next spotlessApply will reformat the string.